lotr-app/pages/volumes/[slug].js

```diff
@@ -0,0 +1,36 @@
+import { useRouter } from "next/router";
+import { volumes } from "@/public/lib/data";
+import Link from "next/link";
+import Image from "next/image";
+
+export default function VolumeDetail() {
+  const router = useRouter();
+  const { slug } = router.query;
+
+  const volume = volumes.find(() => slug === volume.slug);
```

Comment:
Olena, here I wonder if you don't have to pass the current item or volume in the callback function:

const volume = volumes.find((volume) => slug === volume.slug);

I'm not sure if it's really needed